commands/translate.js

```diff
@@ -0,0 +1,18 @@
+const { SlashCommandBuilder } = require('discord.js')
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('translate')
+    .setDescription(' Translate Bla Bla To Leanguage (EN,SY)')
+    .addStringOption((option) => option.setName('word').setDescription('Word to translate').setRequired(true))
+    .addStringOption((option) => option.setName('language').setDescription('Language to translate').setRequired(true)),
+
+  async execute(interaction) {
+    const word = interaction.options.getString('word')
```

Comment:
should add await `interaction.defereReply({ephermal: true})` to avoid [`Unkown interaction`](https://stackoverflow.com/a/68774492) error.